db/migrate/20170103104020_create_course_lectures.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+class CreateCourseLectures < ActiveRecord::Migration
+  def change
+    create_table :course_lectures do |t|
```

Comment:
Should the lectures be disabled in Lesson plan page ?  If we need that, the table should use lesson plan as the parent table, otherwise this is fine.